### PR TITLE
release 6.2.4

### PR DESCRIPTION
**Pull request by gotjosh**
### Bug Fixes
  * **Grafana-CLI**: Fix receiving flags via command line . [#17617](https://github.com/grafana/grafana/pull/17617), [@gotjosh](https://github.com/gotjosh)
  * **HTTPServer**: Fix X-XSS-Protection header formatting. [#17620](https://github.com/grafana/grafana/pull/17620), [@yverry](https://github.com/yverry)

### CODE REVIEW

File: pkg/cmd/grafana-cli/main.go [hunk 1]
```diff
@@ -18,13 +17,13 @@ var version = "master"
...
 	setupLogging()
 
-	flag.Parse()
 	app := cli.NewApp()
 	app.Name = "Grafana cli"
 	app.Usage = ""
 	app.Author = "Grafana Project"
 	app.Email = "https://github.com/grafana/grafana"
 	app.Version = version
+
```
Comment (hugohaggmark) [inline pkg/cmd/grafana-cli/main.go]: Nit-pick: is this change necessary?

**Review by marefr — APPROVED** [review]
LGTM

File: pkg/cmd/grafana-cli/main.go (hunk 1 above)
Comment (gotjosh) [reply pkg/cmd/grafana-cli/main.go]: Sadly 🙈 , this is a cherrypick of an already introduced change. I don't feel comfortable changing anything from this PR without changing it on the master branch.

That being said, I feel like it reads a bit better, but I can see the argument going both ways. 😄
